backend/src/main/kotlin/de/sovity/chatapp/services/edc/EdcSetupService.kt

```diff
@@ -0,0 +1,16 @@
+package de.sovity.chatapp.services.edc
+
+import io.quarkus.runtime.Startup
+import jakarta.enterprise.context.ApplicationScoped
+
+@ApplicationScoped
+class EdcSetupService(
+    private val edcApiService: EdcApiService
+) {
+
+    @Startup
+    fun setupNotificationReceiver() {
+        edcApiService.createNotificationReceiverAssetIfNotExists()
+        edcApiService.createNotificationReceiverDataOfferIfNotExists()
```

Comment:
if you call the service "notificationAssetService" you can just call it "createDataOffer" / "createAsset"